en/releases/README.md

```diff
@@ -1,8 +1,115 @@
 # Releases
 
-A list of PX4 release notes, they contain a list of the changes that went into each release, explaining the included features, bug fixes, deprecations and updates in detail.
+## Release Notes
+
+List of the changes that went into each release, explaining the included features, bug fixes, deprecations and updates in detail.
 
 * [v1.13](../releases/1.13.md)
 * [v1.12](../releases/1.12.md)
 
 The full archive of releases for the PX4 autopilot project can be found on [GitHub](https://github.com/PX4/PX4-Autopilot/releases)
+
+## Release Versioning
+
+PX4 uses a Semantic versioning (`Major.minor.patch[-beta#]`) system ([Wikipedia article](https://en.wikipedia.org/wiki/Software_versioning)).
+
+- Change in `Major`: Breaking changes
+- Change in `minor`: New non-breaking changes
+- Change in `patch`: All other non-breaking changes
+
+## Release Schedule
+
+PX4 follows the following release schedule:
+
+<!-- TO BE DEFINED -->
+
+- `Major` releases: By consensus among Maintainers & Community
+- `minor` releases: Roughly every 6 months
+- `patch` releases: When critical bug-fix / hardware support is needed
+- `beta` releases: Every 1 week during beta testing phase
```

Comment:
I would word it that we do a major or minor release every 6 months. The choice whether it's a major or minor release should be based on whether we change the contract/interface (major) or just add and enhance stuff internally (minor).

Reply:
Done!